l10n_it_ddt/models/sale.py

```diff
@@ -56,13 +56,19 @@ def _compute_ddt_ids(self):
          ('shipping_partner', 'Shipping Partners'),
          ('code_group', 'Code group')], 'DDT invoicing group',
         default='billing_partner')
+    ddt_count = fields.Integer(string='DDTs', compute='_compute_ddt_count')
```

Comment:
```suggestion
    ddt_count = fields.Integer(string='TDs', compute='_compute_ddt_count')
```
ok lasciare il nome della variabile `ddt_..` per coerenza con gli altri campi, ma in inglese il DDT è tradotto DT.

Reply:
Non l'ho fatto per coerenza con le altre stringhe presenti. https://github.com/OCA/l10n-italy/commit/8967c02a420608826a3322da148af3ee0a07fc7d è stato solo fatto per la versione `12.0`.

Reply:
Dovendo aggiungere una nuova stringa, penso sia meglio aggiungerla corretta..
Non so, il nostro Head of Translations @primes2h cosa ne pensa?

Reply:
> Dovendo aggiungere una nuova stringa, penso sia meglio aggiungerla corretta..

Ribadisco quanto ho scritto sopra e aggiungo che, con l'uscita della `14.0` (a breve), la `10.0` non sarà più ufficialmente supportata in virtù di https://github.com/OCA/l10n-italy/issues/1479

A fronte di ciò ritengo inutile investire ulteriore tempo per apportare le modifiche richieste.

Reply:
> Dovendo aggiungere una nuova stringa, penso sia meglio aggiungerla corretta..
> Non so, il nostro Head of Translations @primes2h cosa ne pensa?

Concordo con @tafaRU che in questo caso specifico non ne vale la pena.
Tra l'altro correggere solo queste due stringhe a fronte di molte altre simili presenti nel modulo creerebbe confusione.
La soluzione sarebbe uniformarle tutte aprendo una nuova PR, ma per i motivi esposti da @tafaRU non penso proprio ne valga la pena.
IMHO, meglio dirottare le (limitate) forze sulle versioni supportate più recenti. :-)